src/pages/CurrentWeather/CurrentWeather.jsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from 'react';
+import { ButtonsList } from 'components/ButtonsList/ButtonsList';
+import { InfoDay } from 'components/InfoDay/InfoDay';
+import { fetchCurrentWeather } from 'service/weather-api';
+import { Loading } from 'components/Loader/Loader';
+import { Notification } from 'components/Notification/Notification';
+import helpers from 'Helpers';
+
+const CurrentWeather = () => {
+  const { initialStateCity, Status } = helpers;
+
+  const [city, setCity] = useState(initialStateCity);
+  const [weather, setWeather] = useState([]);
+  const [error, setError] = useState(false);
+  const [status, setStatus] = useState(Status.IDLE);
+
+  useEffect(() => {
+    setStatus(Status.PENDING);
+
+    fetchCurrentWeather(city)
+      .then((data) => {
+        setWeather(data);
+        setStatus(Status.RESOLVED);
+      })
+      .catch((error) => {
+        setError({ message: 'Error not found' });
+        setStatus(Status.REJECTED);
+      });
+  }, [Status.PENDING, Status.REJECTED, Status.RESOLVED, city]);
```

Comment:
your dependencies looks like wrong. You don't need this Status.PENDING, Status.REJECTED, Status.RESOLVED.

Reply:
Initially, they were required by useEffect. useEffect showed this. Now there is no.. Fix